samples/peripheral/radio_test/src/radio_test.c

```diff
@@ -756,6 +767,10 @@ static void radio_rx(uint8_t mode, uint8_t channel, enum transmit_pattern patter
 #endif /* CONFIG_FEM */
 
 	radio_start(true, sweep_processing);
+
+	if (rx_packet_num) {
```

Comment:
By coding guidelines, we should avoid this. Use `if (rx_packet_num > 0)` or `if (rx_packet_num != 0)`

Reply:
Good point.